src/frontends/pytorch/src/op/search_sorted.cpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/op/search_sorted.hpp"
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+using namespace ov::op;
+
+OutputVector translate_search_sorted(const NodeContext& context) {
+    num_inputs_check(context, 2, 5);
+    Output<Node> sorted;
+    Output<Node> values;
+    std::tie(sorted, values) = get_inputs_with_promoted_types(context, 0, 1);
+    const bool out_int32 = context.const_input<bool>(2);
+    PYTORCH_OP_CONVERSION_CHECK(out_int32 == false, "aten::searchsorted(out_int32=true) unsupported");
+    const bool right_mode = context.const_input<bool>(3);
+    PYTORCH_OP_CONVERSION_CHECK(context.input_is_none(4), "aten::searchsorted(side) unsupported");
+    PYTORCH_OP_CONVERSION_CHECK(context.input_is_none(5), "aten::searchsorted(out) unsupported");
```

Comment:
Instead failing conversion with out please consider enabling it, for example like in round op: https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/pytorch/src/op/round.cpp#L27-L29
+extend tests to cover such case:
https://github.com/openvinotoolkit/openvino/blob/master/tests/layer_tests/pytorch_tests/test_round.py#L34-L35

Reply:
Same can be said about `out_i32` which is just a `Convert` to i32

Reply:
The task was to add minimal version that supports some roblox model - not a full scale op. out_i32 parameter and out parameter are not used in that model and as such are omitted. If needed can be always enabled later.

Reply:
That is fine, just please ensure we have a ticket for improved support

Reply:
CVS-155641